examples/09_Prognostic Example.ipynb

```diff
@@ -455,20 +457,23 @@
  ],
  "metadata": {
   "kernelspec": {
-   "display_name": "Python 3.10.7 64-bit",
+   "display_name": ".venv",
```

Comment:
Should this line, and from 465 - 476, be included?

Reply:
Will update - notebook metadata changed based on my kernel